validator/testcases/javascript/actions.py

```diff
@@ -81,16 +78,26 @@ def trace_member(traverser, node, instantiate=False):
         base = trace_member(traverser, node["object"], instantiate)
         base = _expand_globals(traverser, base)
 
+        identifier = _get_member_exp_property(traverser, node)
+
         # Handle the various global entity properties.
         if base.is_global:
             # If we've got an XPCOM wildcard, return a copy of the entity.
             if "xpcom_wildcard" in base.value:
                 traverser._debug("MEMBER_EXP>>XPCOM_WILDCARD")
+
+                from predefinedentities import CONTRACT_ENTITIES
```

Comment:
On Tue, May 05, 2015 at 07:42:16AM -0700, Mathieu Agopian wrote:
>>          # Handle the various global entity properties.
>>          if base.is_global:
>>              # If we've got an XPCOM wildcard, return a copy of the entity.
>>              if "xpcom_wildcard" in base.value:
>>                  traverser._debug("MEMBER_EXP>>XPCOM_WILDCARD")
>> +
>> +                from predefinedentities import CONTRACT_ENTITIES
>
>Why not put that at the top of the module? Import loop? If so, please comment

I'm not sure. There are 6 other local imports of 
`predefinedentities` in this file, and I didn't have the time to 
try to find any side-effects of making them global when I was 
writing this, so I opted to stay on the safe side.